tsconfig.json

```diff
@@ -4,7 +4,9 @@
   "compilerOptions": {
     "baseUrl": "./",
     "paths": {
-      "customerio-reactnative": ["./src/index"]
+      "customerio-reactnative": ["./src/index"],
+      "@segment/analytics-react-native": ["../@segment/analytics-react-native/src/index"],
+      "@segment/sovran-react-native": ["../@segment/sovran-react-native/src/index"]
```

Comment:
Why are these needed in the tsconfig file?

Reply:
This tell the import libraries the path to import libraries from.

Reply:
I do understand that is what the `paths` config option in `tsconfig` is doing. 

I do not understand the reason why we need to modify the `tsconfig` file. 

The 2 segment modules are npm dependencies declared in `package.json`. Therefore, I would expect that our code should be able to import these dependencies via npm without the need to modify `tsconfig` file.

Reply:
I believe its probably because of the fact that they are using a separate `yarn workflow` that sets their tsconfig.

Reply:
Currently, the libraries are being installed from GitHub instead of npm. As a result, the `tsconfig` file does not find the necessary paths defined, hence throwing the errr  `Cannot find module '@segment/analytics-react-native' or its corresponding type declarations`.  Adding the path mappings to the `tsconfig` file resolves this issue and allow the imports to be correctly resolved. I hope this helps.